src/Core/Models/DefaultRights.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Altinn.Platform.Authentication.Core.Models
+{
+    public class DefaultRights
```

Comment:
I think that we can introduce some namesapce/folder structure that can differntiate this "Default Rights" from the generic rights f.ex Altinn.Platform.Authentication.Core.SystemRegister.Models